Lesson34/feyza/lesson34-app/src/components/Account/index.jsx

```diff
@@ -0,0 +1,10 @@
+import { UserDetailsForm } from "../UserDetailsForm";
+
+export const Account = () => {
+  return (
+    <div className="account-container">
+      <h1>Account information</h1>
+      <UserDetailsForm incomingUsername="Placeholder user name" />
```

Comment:
Don't pass 'incomingUsername' as prop, the value should be used from Context instead.